PriceQuoteIosTests/Salesman/SalesmanApiDataSourceTests.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  SalesmanApiDataSourceTests.swift
+//  PriceQuoteIosTests
+//
+//  Created by Livia Vasconcelos on 27/09/20.
+//  Copyright © 2020 Livia Vasconcelos. All rights reserved.
+//
+
+import XCTest
+@testable import PriceQuoteIos
+
+class SalesmanApiDataSourceTests: XCTestCase {
```

Comment:
This might be better to think about what kind of tests you want to have on this project and what you want to achieve. If you really want to go with a BDD-way of testing, you should use Quick/Nimble for this, for organization in general they are the way to go.
If you want to learn more about the XC framework for unit testing, search around for more detailed examples to have a more concise suite and get advantage of the good things that the framework offer (performance, native API, ability to run single tests, etc)

Reply:
Refactored!